esp/esp/program/modules/handlers/volunteersignup.py

```diff
@@ -106,6 +108,35 @@ def volunteerDesc(self):
             base_dict[key] = 'Volunteers for shift "%s"' % req.timeslot.description
         return base_dict
 
+    def volunteerhandout(self, request, tl, one, two, module, extra, prog, template_file='volunteerschedule.html'):
+        #   Use the template defined in ProgramPrintables
+        from esp.program.modules.handlers import ProgramPrintables
+        context = {'module': self}
+        pmos = ProgramModuleObj.objects.filter(program=prog,module__handler__icontains='printables')
+        if pmos.count() == 1:
+            pmo = ProgramPrintables(pmos[0])
+            if request.user.isAdmin() and 'user' in request.GET:
+                volunteer = ESPUser.objects.get(id=request.GET['user'])
+            else:
+                volunteer = request.user
+            scheditems = []
+            offers = VolunteerOffer.objects.filter(user=volunteer, request__program=self.program)
+            for offer in offers:
+                scheditems.append({'name': volunteer.name(),
+                                   'volunteer': volunteer,
+                                   'offer' : offer})
+            #sort the offers by timeslot
+            scheditems.sort(key=lambda item: item['offer'].request.timeslot.start)
+            context['scheditems'] = scheditems
+            return render_to_response(pmo.baseDir()+template_file, request, context)
+        else:
+            raise ESPError('No printables module resolved, so this document cannot be generated.  Consult the webmasters.', log=False)
+
+    @aux_call
+    @needs_account
+    def volunteerschedule(self, request, tl, one, two, module, extra, prog):
```

Comment:
You can just inline this -- since you only use it once, no need to have a separate wrapper.  Just hardcode the `template_file` in the `render_to_response` call.